packages/fury-adapter-swagger/lib/json-schema.js

```diff
@@ -166,6 +166,15 @@ const convertSubSchema = (schema, references, swagger) => {
     actualSchema.type = 'string';
   }
 
+  if (schema.pattern && schema.minLength && schema.pattern.startsWith('^[') && schema.pattern.endsWith(']*$')) {
+    // If a schema has a minimal length (minLength) > 0 AND there is a regex
+    // such as: `^[A-z]*$`, the schema can resolve to an empty string which
+    // doesn't match minLength.
+    //
+    // JSON Schema Faker will fail in that case and get into an infinite loop.
+    actualSchema.pattern = schema.pattern.replace('*$', '+$');
```

Comment:
Is there a way to make the modification closer to the invocation of that faker tool?

Reply:
Unfortuantely not, without copying every schema and traversing over it at the JSON Schema Faker entrypoint. I would like to avoid that, it would be a far more expensive operation.